ansible/openhpc.yml

```diff
@@ -5,36 +5,39 @@
 # Setup ceph mount
 - import_playbook: manila-client.yml
 
-# Setup hyperconverged glusterfs cluster
-- import_playbook: gluster-cluster.yml
-
-# Mount hyperconverged glusterfs volume
-- import_playbook: gluster-client.yml
-
 # Add cluster users
 - import_playbook: users.yml
 
 # Add admin users
-- import_playbook: cluster-users.yml
+- import_playbook: cluster-admins.yml
 
 # Cluster login setup
 - hosts: cluster_login
-  remote_user: centos
   become: yes
   roles:
     - role: openhpc_control
 
 # Cluster batch setup
 - hosts: cluster_batch
-  remote_user: centos
   become: yes
   roles:
     - role: openhpc_compute
 
 # Cluster setup
-- hosts: cluster
-  remote_user: centos
+- hosts: 
+  - cluster_login
+  - cluster_batch
   become: yes
   roles:
     - role: openhpc_runtime
       slurm_partitions: "{{ openhpc_slurm_partitions }}"
+
+# Use mdadm to create software defined raid
+#- import_playbook: mdadm-adm.yml
+
+# Setup hyperconverged glusterfs cluster
+#- import_playbook: glusterfs.yml
+
+# Mount hyperconverged glusterfs volume
+#- import_playbook: beegfs.yml
```

Comment:
I think I quite like the model of composability where we can just run a series of playbooks to build an appliance, orchestrated either through a shell script or a top level playbook full of 'include's.

Reply:
I'll think on it.

Reply:
Alternatively, if these were to be uncommented and run based on group membership, would that work?

Reply:
Could look at that later.